### ISSUE TITLE
Mypy cannot find the Coverage type from version `7.2.0`

### ISSUE THREAD

**Issue by lucas-bremond** [coveragepy/coveragepy#1564]
**Describe the bug**

Mypy complains about not being able to find the `Coverage` attribute from the `coverage` module.

**To Reproduce**

Following the [documentation](https://coverage.readthedocs.io/en/7.2.0/api.html):

```py
import coverage

cov = coverage.Coverage()
```

and applying `mypy` leads to the following error:

```py
error: "Module coverage" does not explicitly export attribute "Coverage"  [attr-defined]
```

However, doing:

```py
import coverage

cov = coverage.control.Coverage()
```

works fine.

**Expected behavior**

Mypy should not complain about the type when Coverage objects are used according to the documentation.

**Additional context**

This problem appeared starting `7.2.0` (using `7.1.0` works fine).

**Comment by nedbat** [comment]
Thanks, clearly I don't fully understand typechecking yet!  I think I've fixed it. Can you install this and give it a try?
```
pip install git+https://github.com/nedbat/coveragepy@nedbat/fix-1564#egg=coverage==0.0
```

**Comment by nedbat** [comment]
This is fixed in c3a3421721788d863012330bc0789b828ce4bcd0.

**Comment by nedbat** [comment]
This is now released as part of [coverage 7.2.1](https://pypi.org/project/coverage/7.2.1).
